release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2733,6 +2733,51 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-23_{context}"]
+=== RHSA-2024:4699 - {product-title} 4.15.23 bug fix and security update
+
+Issued: 2024-07-25
+
+{product-title} release 4.15.23 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4699[RHSA-2024:4699] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4702[RHSA-2024:4702] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.23 --pullspecs
+----
+
+[id="ocp-4-15-23-enhancements_{context}"]
+==== Enhancements
+
+The following enhancement is included in this z-stream release:
+
+[id="ocp-4-15-23-timeout-ingress_{context}"]
+===== Adding connectTimeout tuning option to Ingress Controller API
+
+* The IngressController API is updated with a new tuning option, `ingresscontroller.spec.tuningOptions.connectTimeout`, which defines how long the router will wait for a response when establishing a connection to a backend server. (link:https://issues.redhat.com/browse/OCPBUGS-36208[*OCPBUGS-36208*])
+
+[id="ocp-4-15-23-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the OpenSSL versions for the Machine Config Operator and the hosted  control plane were not the same. With this release, the FIPS cluster `NodePool` resource creation for {product-title} 4.14 and {product-title} 4.15 has been fixed and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37266[*OCPBUGS-37266*])
```

Comment:
```suggestion
* Previously, the OpenSSL versions for the Machine Config Operator and the hosted control plane were not the same. With this release, the FIPS cluster `NodePool` resource creation for {product-title} 4.14 and {product-title} 4.15 has been fixed and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37266[*OCPBUGS-37266*])
```
@cbippley Tiny nit: You have an extra space here. I am not going to block the merge over it, but maybe you want to fix it in a follow-up PR or make a note to fix it with the next z-stream release.

Reply:
Thanks! I'll fix it in the next release. @michaelryanpeter!